src/lib/file-picker/_core.scss

```diff
@@ -0,0 +1,78 @@
+@use './token-utils' as *;
+@use '../core/styles/typography';
+@use '../core/styles/theme';
+@forward './token-utils';
+
+@mixin host() {
+ display: block;
+ height: 100%;
+ width: 100%;
+}
+
+@mixin base() {
+ display: grid;
+ justify-content: center;
+ align-content: center;
+ gap: #{token(gap)};
+
+ box-sizing: border-box;
+ padding: #{token(padding)};
+ height: #{token(height)};
+ width: #{token(width)};
+ max-width: #{token(max-width)};
+ text-align: center;
+ background-color: #{token(background)};
```

Comment:
Not that it matters, but just wanted to mention that we've been using just `background` instead of `background-color` to allow for more options than just color with the shorthand property in the future if needed.